### DISCUSSION TITLE
New ideas to implement in OSWH

### DISCUSSION THREAD

**Discussion by hasnainmakada-99** [hasnainmakada-99/Open-Source-With-Hasnain#105] [Ideas]
Hey @edsodz and @yung-coder would you both like to be a maintainer of OSWH, because I have some new ideas to implement in upcoming months and I cannot do that alone, Let's work in a team shall we ?

**Comment by yung-coder** [comment]
Yeah sure 🚀 @hasnainmakada-99

**Reply by hasnainmakada-99** [reply]
Okay then, Lets wait for @edsodz reply

**Comment by ghost** [comment]
@hasnainmakada-99 let's do it 🚀

**Reply by hasnainmakada-99** [reply]
Sending you the invite @edsodz

**Comment by hasnainmakada-99** [comment]
Hey @edsodz and @yung-coder I've sent you both the invites to the project
